File: ShoppingList2000Backend/Domain/Entities/Product.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class Product
    {
        public string Name { get; set; }
        public bool IsPermanent { get; set; }
        public bool IsChecked { get; set; }
        public int Count { get; set; }
    }
}

File: ShoppingList2000Frontend/src/models/Product.ts
export class Product {
  name: string;
  isPermanent: boolean;
  isChecked: boolean;
  count: number;

  constructor(name: string, isPermanent: boolean, isChecked: boolean, count: number) {
    this.name = name;
    this.isPermanent = isPermanent;
    this.isChecked = isChecked;
    this.count = count;
  }
}
